usefulUtilities/marketplaceItemAdvertiser/marketplaceItemAdvertiser.js

```diff
@@ -0,0 +1,44 @@
+//
+//  marketplaceItemAdvertiser.js
+//
+//  Created by Zach Fox on 2019-02-25
+//  Copyright 2019 High Fidelity, Inc.
+//
```

Comment:
Should there be a small blurb in file for what this does?

Reply:
pointing to the README is sufficient; i don't want to duplicate information